config.yml

```diff
@@ -14,6 +15,28 @@ ska:
   fastq_qual: 20
   fastq_cov: 4
 
+# gubbins options
+gubbins:
+  prefix: gubbins
+  tree-builder: iqtree
+  min-snps: 3
+  min-window-size: 100
+  max-window-size: 10000
+  iterations: 10
+  dir: output/strains/{strain}/
```

Comment:
I don't think this should be an input parameter, it should just be set in the snakefile

Reply:
Also use underscores rather than dashes for consistency

Reply:
oh okay I can take them out. I was under the assumption that we had talked in a meeting to include only a few options but of gubbins but to include `--tree-builder`, all the recombination detection options and `--iterations`. But I can change that

Reply:
Sorry, I just meant `dir` should be in the snakefile. The others are all fine, just change `max-window-size` to `max_window_size` etc